charmcraft/env.py

```diff
@@ -40,6 +41,25 @@ def get_managed_environment_project_path():
     return get_managed_environment_home_path() / "project"
 
 
+def get_managed_environment_snap_channel_default() -> str:
+    """Channel to use when installing Charmcraft snap from Snap Store.
+
+    May someday be updated to use a track known to be compatible with the running
+    version.  For now, we default to default track's stable.
+
+    :returns: Channel string.
+    """
+    return "stable"
```

Comment:
Can't this be just a module's level "constant"?

Reply:
I left it as a function because it may have more complex logic in the future. For example, specifying a track that matches the host Charmcraft version to ensure compatibility, etc.

Reply:
But you're *expecting* for this to be more complex? or just in case? 

If there's no more functionality *planned* to put it there, make it simpler.

Reply:
Done! (Removed get_managed_environment_snap_channel_default())